views/forms/ConfigFields/index.jsx

```diff
@@ -13,12 +13,19 @@ const ConfigFormFields = ({
   onCopy,
   onRemove,
   assessmentOptions,
-  handleClearAssessments,
   handleAssessmentSearch,
+  handleClear,
+  handleClearAssessments,
+  handleSelectAll,
```

Comment:
This is for selecting all users, right? Maybe a more specific name? Not a blocker though